Tigran Kirakosov/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Dosis:200,300,400,600&display=swap" rel="stylesheet">
+  <title>Tokyo Weather Widget</title>
+</head>
+<body>
+  <article class="weather-container_main">
+    <div class="weather-container_widget">
+      <div class="weather-container_first-half">
+        <div class="current-weather__state"><img class="current-weather__state_image" src="/assets/images/icons/sun-ico.png"></div>
+        <div class="current-weather__desc">Sunny</div>
+        <div class="current-weather__date">Tokyo - Wednesday, July 31</div>
+      </div>
+      <div class="weather-container_second-half">
+        <div class="current-weather__temperature">28 &deg;C</div>
+        <div class="current-weather__graph"><img class="current-weather__graph_image" src="/assets/images/graph.png"></div>
+        <div class="current-weather__next-days">

```

Comment:
It looks like list of similar elements - can you use `<ul>` and `<li>` instead of `<div>`?

Reply:
I'll rethink the approach I'm using. Definitely, I should use `ul`/`li` here